Debugger Container fails to build

Logs of the issue is as follows:

```
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 snapd : Depends: systemd but it is not going to be installed
         Depends: udev but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

ERROR: for debugger  (<Service: debugger>, "The command '/bin/sh -c apt-get update && apt-get -y install nano make git kafkacat python python-pip python3-pip python3-dev python3-setuptools ipython3 python-setuptools build-essential nodejs dnsutils virtualenv snapd npm wget apt-transport-https curl apache2-utils imagemagick gettext-base jq nginx' returned a non-zero code: 100")
Service 'debugger' failed to build: The command '/bin/sh -c apt-get update && apt-get -y install nano make git kafkacat python python-pip python3-pip python3-dev python3-setuptools ipython3 python-setuptools build-essential nodejs dnsutils virtualenv snapd npm wget apt-transport-https curl apache2-utils imagemagick gettext-base jq nginx' returned a non-zero code: 100
make: *** [Makefile:338: build] Error 1
```
Will try to update the base image from ubuntu:18.04 to 20.04 to fix the issue.

--- a/docker/debugger/Dockerfile
+++ b/docker/debugger/Dockerfile
@@ -1,6 +1,8 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
 
-RUN apt-get update && apt-get -y install nano make git kafkacat python python-pip python3-pip python3-dev python3-setuptools ipython3 python-setuptools build-essential nodejs dnsutils virtualenv snapd npm wget apt-transport-https curl apache2-utils imagemagick gettext-base jq nginx
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get -y install nano make git kafkacat python python3-pip python3-dev python3-setuptools ipython3 python-setuptools build-essential nodejs dnsutils virtualenv snapd npm wget apt-transport-https curl apache2-utils imagemagick gettext-base jq nginx
 
 # install kubectl
 RUN curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -

